app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -78,7 +78,8 @@ def planning_application_params
                       :applicant_first_name, :applicant_last_name,
                       :applicant_phone, :applicant_email,
                       :agent_first_name, :agent_last_name,
-                      :agent_phone, :agent_email, :questions, :plans]
+                      :agent_phone, :agent_email, :questions, :plans,
+                      :payment_reference]
```

Comment:
I wonder if, longer term, there is a possibility that there will be more than one payment reference? I think for now this is sufficient and we can go ahead and merge, but I wanted to raise the question. Also, do we have a requirement for making this a mandatory field and validating against it? I couldn't see it in the A/Cs but again, just asking.

Reply:
These are two good product questions, but for MVP, I think we should assume that there is only one payment reference, and that it is optional for now.